src/loading/LoadingConsumer.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
```

Comment:
Should be a jsx file. I see that I had it wrong for the snack: `src/snackbar/SnackbarConsumer.js`, could be changed also there.

Reply:
Yeah I just started the branch off Snackbar with shallow copy. Will change them in this PR now